app/src/main/java/io/gnosis/kouban/ui/address/capture/AddressCaptureFragment.kt

```diff
@@ -73,8 +75,10 @@ class AddressCaptureFragment : BaseFragment<FragmentAddressCaptureBinding>() {
                 is Error -> {
                     val stringId = when (viewState.throwable) {
                         is AddressNotSet -> R.string.error_address_not_set
+                        is InvalidSafeAddress -> R.string.error_invalid_safe
                         else -> R.string.error_unknown
                     }
+                    Timber.e(viewState.throwable)
```

Comment:
proposal: we could show logs, timber, stacktraces in viewmodels and ui stuff like snackbars in Fragments/Activities. We have all raw exceptions/throwables in view models anyway.

Reply:
That is definitely worth discussing. I would honestly have a snackbar that also logs to timber as side effect. Logging directly in the viewModel, where the exception is thrown feels a lot like just drowning the exception.